wpa_supplicant/patches/04-bswap.patch

```diff
@@ -0,0 +1,15 @@
+diff -ruN a/src/utils/common.h b/src/utils/common.h
+--- a/src/utils/common.h	2021-05-31 16:31:59.653375453 +0200
++++ b/src/utils/common.h	2021-05-16 11:47:19.467296144 +0200
+@@ -53,6 +53,11 @@
+ }
+ #endif /* __APPLE__ */
+ 
++#ifdef __phoenix__
```

Comment:
the `phoenix` define should be provided by the compiler, so we should use it, see:

https://github.com/gcc-mirror/gcc/blob/9faaa091e53db9cb4aee137bb4a6a9c6a18c7af5/gcc/config/phoenix.h#L23